cmd/kubeadm/app/cmd/token.go

```diff
@@ -220,6 +220,7 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 		}
 	}
 
+	// TODO: Validate usages here so we don't allow something unsupported
```

Comment:
This libe was removed in #55377 by mistake. Added here again and it will be removed by #53929.